cms/static/sass/_lib.scss

```diff
@@ -0,0 +1,9 @@
+// ------------------------------
+// Open edX Certificates: Main Style Compile
+
+// About: third party libraries and dependencies import
+
+
+@import '../../../node_modules/edx-pattern-library/node_modules/bourbon/app/assets/stylesheets/bourbon';
```

Comment:
@dan-f This is a really good point. I asked Brian the same thing when he first imported the Pattern Library, and he said it wasn't possible. You can see his response here:

https://github.com/edx/edx-platform/pull/11307#discussion-diff-51042710

Reading that back, I'm not sure I understand it. It seems like it should be possible to make it work, but I don't want to hold up this work while we investigate other options.